blueocean-core-js/src/js/components/ReplayButton.jsx

```diff
@@ -83,7 +83,7 @@ export class ReplayButton extends Component {
         return (
             <div className={`replay-button-component ${outerClass}`} onClick={(event => stopProp(event))}>
                 <a className={`replay-button ${innerButtonClass}`} title={replayLabel} onClick={() => this._onReplayClick()}>
-                    <Icon size={24} icon="replay" />
+                    <Icon size={24} icon="AvReplay" />
```

Comment:
FWIW I don't care what the names are, so long as there's an example of all of them in the storybooks so we know which names are available :)